src/content/en/2021/caching.md

```diff
@@ -17,4 +20,435 @@ featured_stat_3: TODO
+
+Taking a look at the Median TTL across all resource types, we see that even if we cache a similar percentage in total, there is a much longer cache for mobile, particularly for HTML, audio and video. 
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th colspan="6" scope="col">Median TTL (in days)</th>
+      </tr>
+      <tr>
+        <th scope="col">Type</th>
+        <th scope="col">Desktop</th>
+        <th scope="col">Mobile</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Text</td>
+        <td class="numeric">0.2</td>
+        <td class="numeric">0.2</td>
+      </tr>
+      <tr>
+        <td>XML</td>
+        <td class="numeric">1</td>
+        <td class="numeric">1</td>
+      </tr>
+      <tr>
+        <td>Other</td>
+        <td class="numeric">1</td>
+        <td class="numeric">1</td>
+      </tr>
+      <tr>
+        <td>Video</td>
+        <td class="numeric">4</td>
+        <td class="numeric">8</td>
+      </tr>
+      <tr>
+        <td>HTML</td>
+        <td class="numeric">3</td>
+        <td class="numeric">14</td>
+      </tr>
+      <tr>
+        <td>Audio</td>
+        <td class="numeric">0.2</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>CSS</td>
+        <td class="numeric">30</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>Image</td>
+        <td class="numeric">30</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>Script</td>
+        <td class="numeric">30</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>Font</td>
+        <td class="numeric">365</td>
+        <td class="numeric">365</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>TODO: Caption the table above</figcaption>
+</figure>
+
+When we look at the entirety of cacheable and non-cacheable resources,
+
+{{ figure_markup(
+  image="cacheable-responses.png",
+  caption="Percent of cacheable vs non-cacheable responses.",
+  description="TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=674929574&format=interactive",
+  sheets_gid="812850996",
+  sql_file="ttl.sql"
+) }}
+
+## How do cache TTLs compare to resource age?
+We see that Images and Videos maintained the same average age whether from 1st or 3rd party resources. Images consistently had the same resource age of 2 years, while most Video resources were between 8-52 weeks old.
```

Comment:
Edit: 

We see that Images and Videos maintained the same average age whether from 1st or 3rd party resources. Images consistently had a resource age of 2 years, while most Video resources were between 8-52 weeks old.